applications/mobu/secrets.yaml

```diff
@@ -11,3 +11,6 @@ ALERT_HOOK:
   description: >-
     Slack web hook to which mobu should report failures and daily status.
   if: config.slackAlerts
+GITHUB_WEBHOOK_SECRET:
+  description: >-
+    Github webhook secret for integrated repos
```

Comment:
Could you flesh out the description here a little? This wouldn't tell me where that secret is, what I would need to do to change it, etc. In particular, I think it's important to say where in GitHub this secret is also stored, and to note that the same secret is shared by all environments (which I think is the right thing to do).

Also, I know the current secret naming is all over the place, but I'm trying to slowly move things towards the lowercase-with-dashes naming since I think it's a bit more human-readable and makes it clear that the environment variable may not match the secret (because usually we're prepending the application name). So github-webhook-secret here perhaps?